docs/installation.rst

```diff
@@ -31,7 +31,7 @@ Or add this to your composer.json configuration:
 
 	{
 		"require" : {
-			"FriendsOfCake/crud": "~4.0"
+			"FriendsOfCake/crud": "~4.2"
```

Comment:
@lorenzo, it would be good if you can write a 1 liner explaining why it's dangerous.

Also, I was wondering if there is a way to update the version number in 1 place and auto show up in all the README.md and .rst files